packages/coil-extension/src/popup/components/views/CoilPopupView.tsx

```diff
@@ -19,10 +21,12 @@ export const CoilPopupView = () => {
       <Grid container justify='center' alignItems='center'>
         <div>
           <StatusTypography variant='h6' align='center'>
-            Welcome to Coil
+            {user?.fullName
+              ? `Welcome, ${user.fullName.split(' ')[0]}`
+              : 'Welcome to Coil'}
           </StatusTypography>
           <StatusTypography variant='subtitle1' align='center'>
-            Check out our Discover page to explore Web-Monetized content.
```

Comment:
What's the base of this branch? 
I thought I changed all these instances of Web-Monetized already?

Reply:
The base of this branch is the `tipping-closed-beta`

Reply:
Yeah, I guess I should have been more specific. 
What's the base commit/reference whatever for `tipping-closed-beta` ? 
In any case, I thought I'd sorted all these on main a while back